draft-scenarios/clash/dragon_valley.tex

```diff
@@ -0,0 +1,159 @@
+
+  \item Upon visiting a Grail Field, \textbf{choose one} and then put a Black Cube:
+  \begin{itemize}
+    \item Receive: 10 \svg{gold}, 2 \svg{building_materials}, and 1 \svg{valuables}
+    \item Search (3) \svg{spellpower} twice 
+    \item Search (3) \svg{artifact} twice.
+    %\item Roll 2 \svg{resource} 2 \svg{treasure} and (reroll any \svg{experience}).
+  \end{itemize}
+
+%  \item When visiting a Dragon Utopia: Reshuffle the Neutral \svgunit{golden} and \svgunit{azure} Decks, then draw from \svgunit{azure} until you get 3 Dragons and choose 2 (+1 on impossible difficulty), AND draw from \svgunit{golden} until you get one Dragon.
+  \item Dragon Utopias are guarded by 2 \svgunit{azure} dragons and 2 \svgunit{golden} dragons. 
+  \item Upon flagging Dragon Utopia, \textbf{choose one} and then put a Black Cube:
+  \begin{itemize}
+    \item Recruit one of the defeated Dragons for half its \svg{gold} (rounded down) and its \svg{valuables} cost.
+    \item Receive 15 \svg{gold} AND Search (3) any Artifact Deck.
+  \end{itemize}
+\end{itemize}
+After visiting a Dragon Utopia, roll an Attack Die:
+\begin{itemize}
+  \item[\textbf{-1}] -- Dragons' Curse: The defeated Dragons’ spirits curse the land! All Heroes lose \mbox{1 \svgeven{movement}} for the next Round.
+  \item[\textbf{0}] -- Dragons' Revenge: All players must sacrifice a Unit before the end of the Round; that Unit's health is reduced to 0.
+  \item[ \textbf{+1}] -- Scattered Treasure: All players may immediately spend \mbox{1 \svgeven{movement}} to Search (2) the Artifact Deck.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Suggested Houserules}}
+\begin{itemize}
+  \item \href{https://boardgamegeek.com/thread/3445901/custom-hex-combat-board}{Compact Battlefield}.
+  \item \href{https://boardgamegeek.com/thread/3449937/houserule-for-stacking-more-than-pack}{Stacking more than Pack} - Start with Several \svgunit{bronze} Units instead of Few.
+\end{itemize}
+
+\vspace*{\fill}
+
+\end{multicols*}
+
+\begin{tikzpicture}[overlay]
+  \centering
+  % 2-player
+  \node at (4, -4) {\includegraphics[scale=0.165]{\maps/dragon-valley-2p.png}};
+  \node at (4, -9) {\footnotesize{\textbf{\MakeUppercase{2-PLAYER SCENARIO}}}};
+  % 3-player
+  \node at (13, -4) {\includegraphics[scale=0.165]{\maps/dragon-valley-3p.png}};
+  \node at (13, -9) {\footnotesize{\textbf{\MakeUppercase{3-PLAYER SCENARIO}}}};
+  % 4-player
+  \node at (4, -15) {\includegraphics[scale=0.165]{\maps/dragon-valley-4p.png}};
+  \node at (4, -20) {\footnotesize{\textbf{\MakeUppercase{4-PLAYER SCENARIO}}}};
+  % 4-player short (This map layout is not yet balanced and ready, has to be tested out)
+  % 4-player short. -to be revised. Not tested.
+  \node at (13, -15) {\includegraphics[scale=0.165]{\maps/dragon-valley-4p-short.png}};
+  \node at (13, -20) {\footnotesize{\textbf{\MakeUppercase{4-PLAYER 9 ROUND SCENARIO}}}};
+\end{tikzpicture}
```

Comment:
Please utilize the page real-estate a little more. See how Secret Bomb Stash scenario does this.